app/src/main/java/com/example/workoutapp/MainActivitytwo.kt

```diff
@@ -184,7 +185,7 @@ class MainActivity : ComponentActivity() {
                 content = {
                     Column() {
                         Text(
-                            text = "Hello !",
+                            text = stringResource(id = R.string.Hello),
```

Comment:
Good idea to externalize, but one bigger issue.. Hello ! looks like test text.  Is this even required?